.coveragerc

```diff
@@ -1,3 +1,3 @@
 [run]
-omit = */tests/*, */migrations/*, */settings/*, */wsgi.py, */.venv/*, manage.py, 
-source = .
\ No newline at end of file
+omit = */tests/*, */migrations/*, */settings/*, */wsgi.py, */.venv/*, manage.py, */apps.py, clubs/management/commands/goap_import.py
```

Comment:
I feel like any test that produces less value than the time it takes to create/maintain it should be removed. In this case, many other things would break if `apps.py` was incorrectly configured and this would be caught by the other unit tests. I'm happy to remove this exception and add a simple test, let me know what you think!

Reply:
We already had the test for this written, so un-blacklisting apps.py for now.